openlabcmd/openlabcmd/node.py

```diff
@@ -10,17 +10,14 @@ class NodeStatus(object):
 
 class Node(object):
     def __init__(self, name, role, type, ip, heartbeat=None, alarmed=None,
-                 status=None):
+                 status=None, **kwargs):
         self.name = name
         self.role = role
         self.type = type
         self.ip = ip
-        if not heartbeat:
-            self.heartbeat = 0
-        if not alarmed:
-            self.alarmed = False
-        if not status:
-            self.status = NodeStatus.INITIALIZING
+        self.heartbeat = heartbeat if heartbeat else 0
```

Comment:
`= heartbeat or 0`

Reply:
can the `heartbeat` be initialized with 0 ? that may cause `if heartbeat` clause to be `False`

Reply:
the type of heartbeat  is int. `heartbeat=0` means the service is just initialized.